### PR TITLE
publish account deletion events

### PR DESCRIPTION
**Pull request by nicl**
Adds a general notification to an SNS topic for other teams to consume - thinking particularly of Avatar and Discussion services here which need this for GDPR work.

This is not ready for review, but I would like help on testing and deploying this.

### CODE REVIEW

**Comment by jfsoul** [comment]
cc @mario-galic

File: StepFunctionCloudFormation.yaml [hunk 1]
```diff
@@ -142,6 +142,24 @@ Resources:
               "DeleteUser": {
                 "Type": "Task",
                 "Resource": "arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:deleteUser",
+                "Next": "Notify"
+              },
+              "Notify": {
```
Comment (nicl) [inline StepFunctionCloudFormation.yaml]: NotifyTopic?

File: StepFunctionCloudFormation.yaml [hunk 2]
```diff
@@ -403,6 +421,20 @@ Resources:
         MemorySize: "512"
         Timeout: "20"
 
+  notify:
```
Comment (nicl) [inline StepFunctionCloudFormation.yaml]: Need also to create SNS topic in cfn

**Comment by QuarpT** [comment]
Do we want to publish to the topic on the manual flow?

**Comment by QuarpT** [comment]
Updated the PR, is working
picture 12

**Pull request merged**